code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -286,7 +285,7 @@ var/global/list/damage_icon_parts = list()
 	else
 		//BEGIN CACHED ICON GENERATION.
 		var/obj/item/organ/external/chest = get_organ("chest")
-		base_icon = chest.get_icon()
+		base_icon = chest.get_icon(skeleton)

```

Comment:
Wouldn't that mean you could make skeleton people clones with the brain_scan mode